src/components/browse/sidebar/sidebar.tsx

```diff
@@ -84,26 +85,40 @@ const Sidebar = (props: sidebarProps) => {
       {props.facetList === null || props.facetList.length < 1 ? null : (
         <div className="bg-light border p-2 rounded-3 pt-3">
           <Row className="position-relative w-100 px-0 mx-0">
-            {props.facetList.sort((a, b) => b.key < a.key ? 1 : -1).map((facet, index) => (
-              <Filter
-                facet={facet}
-                key={index}
-                check={check}
-                setCheck={setCheck}
-                setFilterDict={props.setFilterDict}
-                searchKeyword={props.searchKeyword}
-                filterDict={props.filterDict}
-              />
-            ))}
+            {props.facetList
+              .sort((a, b) => (b.key < a.key ? 1 : -1))
+              .map((facet, index) => (
+                <Filter
+                  facet={facet}
+                  key={index}
+                  check={check}
+                  setCheck={setCheck}
+                  setFilterDict={props.setFilterDict}
+                  searchKeyword={props.searchKeyword}
+                  filterDict={props.filterDict}
+                />
+              ))}
           </Row>
           <Row className="mb-2 mt-3">
             <Col className="offset-2" xs md lg={4}>
```

Comment:
Can the button size for both clear and filter be made same? either lg={4} or lg={6}?